feedback/forms.py

```diff
@@ -181,3 +182,38 @@ class ContextTagForm(forms.ModelForm):
     class Meta:
         model = ContextTag
         fields = ['question_key', 'response_value', 'color', 'content']
+
+
+class ImportTagsForm(forms.Form):
+    source_course = forms.ModelChoiceField(queryset=None, label=_("Source course"))
+
+    def __init__(self, target_course: Course, visible_courses: list[int], user, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.target_course = target_course
+        course_options = Course.objects.using_namespace_id(target_course.namespace).exclude(id=self.target_course.id)
+        # Logic copied from feedback/views.py:297
+        if not (user.is_superuser or user.is_staff):
+            course_options = course_options.filter(id__in=visible_courses)
+        self.fields['source_course'].queryset = course_options
+
+    def copy_tags(self):
+        source_course = self.cleaned_data["source_course"]
+        source_tags = source_course.tags.all()
+        target_tags = self.target_course.tags.all()
+        existing_tags = source_tags.filter(slug__in=target_tags.values_list('slug', flat=True))
```

Comment:
Indeed typically if there already exists a tag with the same tag slug, they probably might have the same name (although they might not), but also they may have a different description (or one may not have a description at all), or a different color. In these types of "conflicts", I would assume that it would be nice for the usability if the user could select whether to overwrite the tag with the to-be-imported tag, to not import it at all, or if to rename the slug and still import it. However, this may be somewhat difficult to implement, and probably is acceptable now to implement this in this fashion (so importing duplicate is reduced, especially if tags are usually imported before any new tags are created) and then later on change this if there is need for the improved changes.